docker-compose.yml

```diff
@@ -18,14 +18,18 @@ services:
     command:
       - "--memory.max-traces"
       - "10000"
+    environment:
+      - COLLECTOR_OTLP_ENABLED=true
     ports:
       - "16686:16686"                    # Jaeger UI
       - "14250"                          # Jaeger model.proto endpoint
```

Comment:
Should I also remove this port from here?
Or it makes sense leaving it for now?

Reply:
I think we should remove to avoid confusion.